src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js

```diff
@@ -8,15 +8,6 @@
  */
 
 /* @flow */
-function removeNextBr(parent, component: ?Element) {
-  while (component != null && component.tagName.toLowerCase() !== 'br') {
-    component = component.nextElementSibling;
-  }
-  if (component != null) {
-    parent.removeChild(component);
-  }
-}
-
 function absolutifyCaret(component: Node) {
```

Comment:
you are right, no additional logic remains. Removed the file and cleaned up. Thx for the detailed review.